helm/hpcc/values.schema.json

```diff
@@ -327,6 +327,11 @@
           "description": "which planes (if any) the data is replicated onto (primarily bare metal)",
           "type": "array",
           "items": { "type": "string" }
+        },
+        "labels": {
+          "description": "a list of labels associated with this plane, e.g. lz, data",
```

Comment:
should we also include dll, dali?

Reply:
my thinking was that specific planes, like those two, that they will be named explicitly (as they currently have to be by dllStorage and dataStorage), Dali's could be a property/config option of itself, as it is with the sasha services.

So not sure need additional labels, where named explicitly, and/or can only be 1 (dll).